src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Editor/LoadedProjectFileEditorFactory.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Microsoft.VisualStudio.Shell.Interop;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using IOLEProvider = Microsoft.VisualStudio.OLE.Interop.IServiceProvider;
+using System.Runtime.InteropServices;
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Editor
+{
+    [Guid(EditorFactoryGuid)]
+    class LoadedProjectFileEditorFactory : IVsEditorFactory
+    {
+        public const string EditorFactoryGuid = "da07c581-c7b4-482a-86fe-39aacfe5ca5c";
+        public static readonly Guid XmlEditorFactoryGuid = new Guid("{fa3cd31e-987b-443a-9b81-186104e8dac1}");
```

Comment:
This needs to be refactored into a common area, other things shouldn't be depending on the LoadedProjectFileEditorFactory for this Guid. It's also defined elsewhere.